src/helpers/feature-flags.ts

```diff
@@ -0,0 +1,79 @@
+import { homedir } from "node:os";
+import { join } from "node:path";
+
+interface CachedFeatureFlag {
+  value: boolean;
+  expiresAt: number;
+}
+
+interface FeatureFlagCache {
+  [key: string]: CachedFeatureFlag;
+}
+
+const ONE_DAY_MS = 24 * 60 * 60 * 1000;
+
+function getFeatureFlagCachePath(): string {
+  const configDir = join(homedir(), ".sfcompute");
+  return join(configDir, "feature-flags");
+}
+
+export async function saveFeatureFlags(flags: FeatureFlagCache): Promise<void> {
+  const cachePath = getFeatureFlagCachePath();
+  const configDir = join(homedir(), ".sfcompute");
+
+  try {
+    await Deno.mkdir(configDir, { recursive: true });
+    await Deno.writeTextFile(cachePath, JSON.stringify(flags, null, 2));
+  } catch (error) {
+    console.error("boba error saving feature flags:", error);
+  }
+}
+
```

Comment:
@JohnPhamous @Flaque one of you should rebase this branch on main, I believe this change is outdated.